core/file.rbs

```diff
@@ -1042,8 +1042,6 @@ class File::Stat < Object
 
   def rdev_minor: () -> Integer
 
-  def read: (?int? length, ?string outbuf) -> String?
-
```

Comment:
`File::Stat#read` does not seem to exist.

I removed the definition of this method as it seemed unneeded.

It seems to have been inherited from the Sorbet RBI file from which this file originated.

https://github.com/sorbet/sorbet/blob/a0c88fe9ce2ce713a1e0b386460704284d50c568/rbi/core/file.rbi#L1822-L1829